changelog/58939.added

```diff
@@ -0,0 +1,23 @@
+Salt now supports loading entrypoints by passing a module instead of a function.
```

Comment:
This is probably excessive for the changelog. It will already link to the associated issue, which is where we should put this, plus a link to the actual documentation.